slf4j-jdk-platform-logging/src/main/java/module-info.java

```diff
@@ -0,0 +1,5 @@
+module org.slf4j.jdk {
+    requires org.slf4j;
+    provides java.lang.System.LoggerFinder
+            with org.slf4j.jdk.SLF4JSystemLoggerFinder;
+}
```

Comment:
Should this be in `src/java9`? Would be more consistent but then again, the entire thing only works on Java 9+.

Reply:
java.lang.System.LoggerFinder was introduced in Java9. Thus there would be no point in a multi-release jar.